### PR TITLE
Style

### PR DESCRIPTION
**Pull request by ApaeP**
Added some style:
 - remove the `#` before batch numbers
 - Font menu as bold
 - Safe @day_team if no teacher is on duty
 - dig instead of chaining key calls on hash
 - Add default values for batch status
 - Add some emojis for the lolz

### CODE REVIEW

File: kitt_bar_app/current_batch.rb [hunk 1]
```diff
@@ -40,20 +40,20 @@ def header
...
     return unless @ticket
 
     url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
-    puts "- Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
+    puts "✅ Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
   end
 
   def day_team
-    return if @api_data['on_duties'].empty?
+    return if @api_data['on_duties']&.empty? || @api_data['on_duties'].nil?
```
Comment (paultursuru) [inline kitt_bar_app/current_batch.rb]: Do we want to .dig() here too ?

File: kitt_bar_app/current_batch.rb [hunk 2]
```diff
@@ -87,8 +87,8 @@ def user_ticket
   end
 
   def parse_batch_status
-  	@color 			 	= @api_data['camp']['color'] == "grey" ? "gray" : @api_data['camp']['color']
-  	@ticket_count = @api_data['tickets'].count
-  	@lunch_break  = @api_data['camp']['on_lunch_break']
+  	@color 			 	= @api_data.dig('camp', 'color') == "grey" ? "gray" : @api_data.dig('camp', 'color') || 'gray'
+  	@ticket_count = @api_data['tickets']&.count || -1
```
Comment (paultursuru) [inline kitt_bar_app/current_batch.rb]: Same here ? .dig ?

File: kitt_bar_app/current_batch.rb [hunk 3]
```diff
@@ -16,11 +16,11 @@ def initialize(attr = {})
...
   def menu
 	  puts "---"
-    puts "#{menu_name}"
+    puts "#{menu_name}|font=bold"
     end_ticket
     tickets
-	  puts "- Calendar|href=#{calendar_url}|size=12"
-	  puts "- Students|href=#{classmates_url}|size=12"
+	  puts "🗓 Calendar|href=#{calendar_url}|size=12"
+	  puts "🧑‍🎓 Students|href=#{classmates_url}|size=12"
```
Comment (paultursuru) [inline kitt_bar_app/current_batch.rb:23]: chic !

File: kitt_bar_app/current_batch.rb (hunk 1 above)
Comment (ApaeP) [reply kitt_bar_app/current_batch.rb]: Agreed, we should use dig consistently 👍

**Review by paultursuru — APPROVED** [review]

**Pull request merged**